recipe/meta.yaml

```
@@ -4,57 +4,57 @@
 {% set name = "at-spi2-atk" %}
 {% set version = "2.34.2" %}
 {% set version_majmin = '.'.join(version.split('.', 2)[:2]) %}
-{% set sha256 = "901323cee0eef05c01ec4dee06c701aeeca81a314a7d60216fa363005e27f4f0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: http://ftp.gnome.org/pub/gnome/sources/{{ name }}/{{ version_majmin }}/{{ name }}-{{ version }}.tar.xz
-  sha256: {{ sha256 }}
+  url: https://download.gnome.org/sources/{{ name }}/{{ version_majmin }}/{{ name }}-{{ version }}.tar.xz
+  sha256: 901323cee0eef05c01ec4dee06c701aeeca81a314a7d60216fa363005e27f4f0
 
 build:
-  number: 0
-  skip: True  # [win]
+  number: 1
+  skip: true  # [win]
   run_exports:
     - {{ pin_subpackage('at-spi2-atk', max_pin='x') }}
   missing_dso_whitelist:  # [linux]
     - $RPATH/libpthread.so.0  # [linux]
     - $RPATH/libc.so.6  # [linux]
+
 requirements:
   build:
     - meson
     - ninja
     - pkg-config
     - pthread-stubs  # [linux]
     - {{ compiler('c') }}
-    - {{ cdt('libx11-devel') }}  # [linux]
-    - {{ cdt('libxi-devel') }}  # [linux]
-    - {{ cdt('libxfixes-devel') }}  # [linux]
-    - {{ cdt('libxtst-devel') }}  # [linux and not ppc64le]
-    - {{ cdt('xorg-x11-proto-devel') }}  # [linux]
   host:
     - at-spi2-core
     - atk
     - dbus
     - glib
     - gobject-introspection
+    - xorg-libx11
+    - xorg-libxi
+    - xorg-libxfixes
+    - xorg-libxtst
+    - xorg-xorgproto
```

```suggestion
    - xorg-libx11  # [linux]
    - xorg-libxi  # [linux]
    - xorg-libxfixes  # [linux]
    - xorg-libxtst  # [linux]
    - xorg-xorgproto  # [linux]
```
Should keep them Linux-only, right?

---

Fixed

---

[xorg-* packages brought through at-spi2-core](https://github.com/AnacondaRecipes/at-spi2-atk-feedstock/pull/2/commits/d609834cc60706b12e5b103b6b47e23c23c3f1df)